src/images/loading.svg

```diff
@@ -1,6 +1,6 @@
 ﻿<svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 100% 100%">
-  <text fill="%23333333" x="50%" y="50%" font-family="\'Open Sans\', \'Helvetica Neue\', Helvetica, Arial, sans-serif"
+  <text fill="#333333" x="50%" y="50%" font-family="\'Open Sans\', \'Helvetica Neue\', Helvetica, Arial, sans-serif"
```

Comment:
I'm not sure what this does, is it required if you've created a separate `loading-white.svg`?

Reply:
This just replaces the %23 code with the # as we aren't using the SVGs with fragment identifiers that i can see. I can edit both of them so they use %23 if preferred?